src/components/OptionListContextProvider.tsx

```diff
@@ -64,22 +64,11 @@ function OptionsListContextProvider({reports, children}: OptionsListProviderProp
             return;
         }
 
-        const lastUpdatedReport = ReportUtils.getLastUpdatedReport();
-
-        if (!lastUpdatedReport) {
-            return;
-        }
-
-        const newOption = OptionsListUtils.createOptionFromReport(lastUpdatedReport, personalDetails);
-        const replaceIndex = options.reports.findIndex((option) => option.reportID === lastUpdatedReport.reportID);
-
-        if (replaceIndex === -1) {
-            return;
-        }
+        const newReports = OptionsListUtils.createOptionList({}, reports).reports;
```

Comment:
Why we are passing empty object for the personal details? The personal details are used to get the correct icons. This is causing https://github.com/Expensify/App/issues/42232. While testing I also found that the list becomes irresponsive and get into infinite loop (could be related to change here too?)